### PR TITLE
Turn anderson acceleration off by default

### PR DESCRIPTION
**Pull request by akshayka**
And increase max_iters to 10k

### CODE REVIEW

File: diffcp/cone_program.py [hunk 1]
```diff
@@ -251,10 +251,17 @@ def solve_and_derivative_internal(A, b, c, cone_dict, warm_start=None,
         data["s"] = warm_start[2]
 
     kwargs.setdefault("verbose", False)
+    kwargs.setdefault("acceleration_lookback", 0)
+    kwargs.setdefault("max_iters", 10000)
     result = scs.solve(data, cone_dict, **kwargs)
 
-    # check status
     status = result["info"]["status"]
+    if status == "Solved/Inaccurate" and "acceleration_lookback" not in kwargs:
```
Comment (sbarratt) [inline diffcp/cone_program.py]: Doesn't `setdefault` above always add `acceleration_lookback` to `kwargs`? I'm not sure the code in this block does what you want.

**Comment by akshayka** [comment]
You’re right, I meant to delete the above lines ... will fix

File: diffcp/cone_program.py [hunk 2]
```diff
@@ -250,11 +250,18 @@ def solve_and_derivative_internal(A, b, c, cone_dict, warm_start=None,
...
+
     kwargs.setdefault("verbose", False)
     result = scs.solve(data, cone_dict, **kwargs)
 
-    # check status
     status = result["info"]["status"]
+    if status == "Solved/Inaccurate" and "acceleration_lookback" not in kwargs:
+        # anderson acceleration is sometimes unstable
+        kwargs.setdefault("acceleration_lookback", 0)
+        kwargs.setdefault("max_iters", 10000)
```
Comment (sbarratt) [inline diffcp/cone_program.py]: Should be:
```
kwargs['acceleration_lookback'] = 0
kwargs['max_iter'] = 10000
```

**Comment by sbarratt** [comment]
this is until the issue https://github.com/cvxgrp/scs/issues/129 is fixed

**Pull request merged**